packages/dev/sharedUiComponents/src/nodeGraphSystem/interfaces/portData.ts

```diff
@@ -54,6 +54,7 @@ export interface IPortData {
     hasEndpoints: boolean;
     endpoints: Nullable<IPortData[]>;
     directValueDefinition?: IPortDirectValueDefinition;
+    allowMultipleConnections?: boolean;

```

Comment:
This won't work. We do not support multiple connection to one input. But look at GradientBlock in NPE. this is the way to go